src/module.ts

```diff
@@ -179,6 +179,16 @@ import {
     CONTEXT_STORE,
     CYCLE_COUNTERS
 } from './globals';
+export { isAudioBufferSourceNode } from './guards/audio-buffer-source-node';
+export { isAudioNodeOutputConnection } from './guards/audio-node-output-connection';
+export { isAudioNode } from './guards/audio-node';
+export { isAudioWorkletNode } from './guards/audio-worklet-node';
+export { isBiquadFilterNode } from './guards/biquad-filter-node';
+export { isConstantSourceNode } from './guards/constant-source-node';
+export { isDelayNode } from './guards/delay-node';
+export { isGainNode } from './guards/gain-node';
+export { isOscillatorNode } from './guards/oscillator-node';
+export { isStereoPannerNode } from './guards/stereo-panner-node';
```

Comment:
While I did not see this pattern in the source code, it's a neat way to re-export something without explicitly importing only to do export again.